lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js

```diff
@@ -21,6 +23,7 @@ class InstructionSlide extends React.Component {
             <div className="container__row">
               <div className="container__block -narrow -centered">
                 <h2 className="heading -gamma">{title}</h2>
+                <p>{competition.joined ? competition.messages.confirmation : ''}</p>
```

Comment:
Looks good. But do we want the default to be something related if the message is not present instead of just empty string?